resources/views/pages/select_tenant.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layouts.page_no_menu')
+
+@section('title', __('Seleziona pubblica amministrazione'))
+
+@section('content')
+    <div class="mb-5">
+        @component('layouts.components.box', ['classes' => 'rounded'])
+            <div class="col-lg-12 ml-auto">
+                @include('layouts.includes.public_administration_selector')
+            </div>
+        @endcomponent
+        @if ($hasTenant)
```

Comment:
Come mai è visualizzato un bottone quando esiste la chiave `tenant_id` nella sessione? Il componente select non dovrebbe effettuare un redirect lato client?

Reply:
Il componente select attualmente ricarica la pagina passando il parametro in query, non fa nessun redirect.

Reply:
Si era quello che intendevo con l’espressione “redirect lato client”.